src/components/DataCollector/index.js

```diff
@@ -12,8 +12,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const CONFIG_DOC_URI =
-  "https://www.adobe.io/apis/experienceplatform/home/services/web-sdk.html#!api-specification/markdown/narrative/edge/fundamentals/executing-commands.md";
+const CONFIG_DOC_URI = "https://adobe.ly/2r0uUjh";
```

Comment:
NICE!!! We need some WIKI explaining how to access this tool and edit those links, please.

Voila: https://wiki.corp.adobe.com/display/DMSArchitecture/How+to+edit+short+links